src/vm/eetwain.cpp

```diff
@@ -5729,6 +5729,7 @@ BOOL EECodeManager::IsInFilter(GCInfoToken gcInfoToken,
 }
 
 
+#if defined(_TARGET_X86_) && !defined(WIN64EXCEPTIONS)
```

Comment:
Shouldn't this be just `#ifndef WIN64EXCEPTIONS` to match what's in the header file?